libtsuba/src/tsuba.cpp

```diff
@@ -205,6 +207,108 @@ tsuba::ListAvailableViews(
   return std::make_pair(latest_version, views_found);
 }
 
+katana::Result<std::vector<std::pair<katana::Uri, katana::Uri>>>
+tsuba::CreateSrcDestFromViewsForCopy(
+    const std::string& src_dir, const std::string& dst_dir, uint64_t version) {
+  std::vector<std::pair<katana::Uri, katana::Uri>> src_dst_files;
+
+  // List out all the files in a given view
+  auto rdg_views = KATANA_CHECKED(tsuba::ListAvailableViews(src_dir, version));
+  for (const auto& rdg_view : rdg_views.second) {
+    KATANA_LOG_WARN("view_path: {}", rdg_view.view_path);
+    auto uri = KATANA_CHECKED(katana::Uri::Make(src_dir));
+
+    auto rdg_manifest_res =
+        tsuba::RDGManifest::Make(uri, rdg_view.view_type, version);
+    if (!rdg_manifest_res) {
+      continue;
+    }
+
```

Comment:
Just add `RDGManifest rdg_manifest = std::move(rdg_manifest_res.value())` here so you can refer to this object directly rather than through `value()` all the time.